app/models/manageiq/providers/ibm_power_vc/cloud_manager/event_parser.rb

```diff
@@ -1,10 +1,2 @@
 module ManageIQ::Providers::IbmPowerVc::CloudManager::EventParser
```

Comment:
Are you thinking we might need to override some of the parsing for PowerVC? If not I don't think there's a need to keep the empty module definition.

Reply:
We shouldn't leave unused modules around, if/when you want to override event_parser then we can add this module

Reply:
Yeah, removed the `event_parser` and got it working